app/src/main/res/values/strings.xml

```diff
@@ -11,7 +11,7 @@
     <string name="sample_coords" translatable="false">55.332244 34.442211</string>
     <string name="sample_message" translatable="false">hey I found the cache, it is over here next to the big tiger. I\'m kinda scared.</string>
 
-    <string name="unknown_node_short_name" translatable="false">\???</string>
+    <string name="unknown_node_short_name">UNK</string>
```

Comment:
the short name has a limit of 4 bytes. that is why it is using `?` and is non-translatable.